plonk/Cargo.toml

```diff
@@ -11,13 +11,13 @@ jf-utils = { path = "../utilities" }
 jf-rescue = { path = "../rescue" }
 
 ark-std = { version = "0.3.0", default-features = false }
-ark-serialize = { version = "0.3.0", default-features = false }
-ark-ff = { version = "0.3.0", default-features = false, features = ["asm", "parallel"] }
-ark-ec = { version = "0.3.0", default-features = false, features = ["parallel"] }
-ark-poly = { version = "0.3.0", default-features = false,  features = ["parallel"] }
-ark-bn254 = { version = "0.3.0", default-features = false, features = ["curve"] }
-ark-bls12-377 = { git = "https://github.com/arkworks-rs/curves", features = ["curve"], rev = "677b4ae751a274037880ede86e9b6f30f62635af" }
-ark-bls12-381 = { version = "0.3.0", default-features = false, features = ["curve"] }
+ark-serialize = "0.3.0"
```

Comment:
iiuc. this means we import ark-serialize etc. with default features, is that right?

Reply:
yes, I checked these, default features is no_std. ([see here](https://github.com/arkworks-rs/algebra/blob/master/serialize/Cargo.toml#L27))

The slightly confusing part is that, except `ark-std`, most of `ark-*` are no_std by default.